tp/src/addressing.rs

```diff
@@ -25,9 +25,6 @@ const CONTRACT_REGISTRY_PREFIX: &str = "00ec01";
 /// The contract prefix for global state (00ec02)
 const CONTRACT_PREFIX: &str = "00ec02";
 
-/// The smart permission prefix for global state (00ec03)
```

Comment:
Commit message says "This removes the smart contract from the transaction processor", did you mean to say smart permissions instead of smart contract?

Reply:
Fixed